react/src/constants/routes.js

```diff
@@ -1,5 +1,7 @@
 const routes = {
-  HOME: '/'
-}
+  HOME: '/',
+  DETAIL: '/detail/:id',
+  DETAIL_NO_ID: '/detail/'
```

Comment:
We may export `DETAIL` as a function:
```
DETAIL: (id = ':id') => `/detail/${id}`
```
So:
```
routes.DETAIL() === '/detail/:id'
routes.DETAIL(4) === '/detail/5'
```
The only drawback is that `HOME` is not a function and `DETAIL` is, leading to a inconsistency between these.
Would you like to treat `HOME` as a function too ? What do you think ?